python/py-avro/Portfile

```diff
@@ -1,13 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        apache avro 1.9.1 release-
+name                py-avro
+version             1.10.0
 revision            0
-name                py-${github.project}
-categories-append   devel
+categories-append   python devel
```

Comment:
the `python` category is already added by the `python` PG so please don't append that again

Reply:
Thanks done.